hack/ci/Dockerfile.ci

```diff
@@ -1,10 +1,10 @@
-FROM quay.io/kubevirt/kubevirt-cdi-bazel-builder:0.0.10 AS builder
+FROM quay.io/kubevirt/kubevirt-cdi-bazel-builder:2203231247-92ab271e AS builder
 WORKDIR /go/src/github.com/kubevirt.io/containerized-data-importer
 COPY . .
 
 ENV KUBEVIRT_RUN_UNNESTED=true
 
-RUN microdnf install make docker && \
+RUN dnf -y install make podman && \
```

Comment:
imo worth to install buildah and qemu-user-static (to allow cross compilation with buildah)

Reply:
We don't really use this within CDI but it is supposed to be used for a lane that runs CDI tests on openshift, I avoided making unnecessary changes - just what is needed to make it build (seems like it was broken).